pkg/maintainer/spv/spv.go

```diff
@@ -430,7 +424,7 @@ func getProofInfo(
 	// The other case could be that the transaction is older than the last two
 	// Bitcoin difficulty epochs. In that case the transaction will soon leave
 	// the sliding window of recent transactions.
-	return false, 0, nil
+	return false, accumulatedConfirmations, 0, nil
```

Comment:
Shouldn't we return `0` instead of `accumulatedConfirmations` in this case?

Reply:
I also thought about, but in my opinion if we ended up here, it's not really an error.
It means that the transaction SPV proof goes into an epoch that is not proven in the relay yet.
I'm ok with returning `0`, though. It will not affect the flow.

Reply:
Done in 31c0ce911678b2254f63e7d1ea00b345e0d25a2d.